justfile

```diff
@@ -15,10 +15,10 @@ test:
     pytest -n 2 --cov --random-order
 
 # Run linter and formatter
-fix: (lint) (format)
+fix: (lint) (format) (pre-commit-fix)
 
 # Run lint, format, and type checks
-check: (lint-check) (format-check) (type-check)
+check: (lint-check) (format-check) (type-check) (pre-commit-check)
```

Comment:
- "check-toml"
-  "check-yaml"
-  "check-toml"

All three are checked twice in the pipeline. Once in the `pre-commit` job and again in the `check` job.

The pipeline passes ONLY because the pre-commit job runs before the check job.

Running the same checks twice feels icky. And if it feels icky, it should probably be avoided. Maybe there's a way to run only select commands from justfile? Or create a separate command with the pre-commit checks?

Reply:
Also check-toml is there twice. It should be `check-json`